src/app/test/fakes/Harvest.ts

```diff
@@ -53,9 +53,7 @@ export function generateHarvest(data?: Partial<IHarvest>): Required<IHarvest> {
     uploadPassword: modelData.internet.password(),
     uploadUser: modelData.internet.userName(),
     uploadUrl: `sftp://${modelData.random.word()}.${modelData.random.word()}.org:2020`,
-    mappings: Array(modelData.datatype.number({ min: 0, max: 3 })).map(() =>
-      generateHarvestMapping()
-    ),
+    mappings: Array(generateHarvestMapping()),
```

Comment:
is there a reason for this change? It seems unrelated to the work you were doing

Reply:
The mapping attribute was causing troubles for me when I was developing my tests
This change is no longer needed since the tests are now complete

I have since removed the testing artefact